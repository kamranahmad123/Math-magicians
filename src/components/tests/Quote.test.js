import renderer from 'react-test-renderer';
import DisplayQuotes from '../Quote';

describe('Test quotes', () => {
  test('test if quotes page really matches', () => {
    const quotesRender = renderer
      .create(<DisplayQuotes />)
      .toJSON();
    expect(quotesRender).toMatchSnapshot();
  });
});
