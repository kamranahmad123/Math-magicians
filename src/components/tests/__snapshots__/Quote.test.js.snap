// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test quotes test if quotes page really matches 1`] = `
<div
  className="Quote"
/>
`;
